### DISCUSSION TITLE
CMI - Item repair - EXP cost

### DISCUSSION THREAD

**Discussion by Victor75007** [Zrips/CMI#4029] [Feature Requests]
Hello,

Can you add EXP cost to /repair command?

---
### Detailed explanation of what it should do, when and in what situation: ###
Just simple settings in the config.yml file.

```
    CostsEXP: true
    BaseEXPprice: 80.0  #approx 5 levels
```

### Basic explanation as to why this feature is needed and/or useful to have: ###
Those server owners, who run a "semi-vanilla" server without Economy-system, they're not need install another repair plugin for EXP cost.

**Comment by CloudeLecaw** [comment]
You can create an custom alais where u can define the exp cost. Its Import that u defibe the command from conaile too and dont give permission to the regular cmi repar command.

```
   repair:
    Cmds:
    - perm:cmi.command.customalias.repair! expcost:80#! asConsole! cmi repair [playerName]
```
Dont forget to disable /repair in alias.yml
Permission to use it is: cmi.command.customalias.repair

The command will only be preformed from console if player has the permission and has the defined amount of xp (80xp in this case)
